tutorials/finetune_lora.md

```diff
@@ -164,13 +164,17 @@ python scripts/merge_lora.py \
   --out_dir "out/lora_merged/stablelm-base-alpha-3b/"
 ```
 
-After merging, we can use the `base.py` file for inference using the new checkpoint file. Note that if your new checkpoint directory is different from the original checkpoint directory, we also have to copy over the `*.json` files which are required for the configuration and tokenizer information:
+After merging, we can use the `base.py` file for inference using the new checkpoint file. Note that if your new checkpoint directory is different from the original checkpoint directory, we also have to copy over the `*.json`  files which are required for the configuration and tokenizer information:
 
 ```bash
 cp checkpoints/stabilityai/stablelm-base-alpha-3b/*.json \
 out/lora_merged/stablelm-base-alpha-3b/
 ```
 
+> [!NOTE]
+> Some models also have a `tokenizer.model` file that needs to be copied.
+> For example, `cp checkpoints/meta-llama/Llama-2-7b-hf/{*.json,tokenizer.model} out/lora_merged/Llama-2-7b-hf/`
+
```

Comment:
This should cover both cases

```suggestion
```bash
cp checkpoints/stabilityai/stablelm-base-alpha-3b/tokenizer* \
out/lora_merged/stablelm-base-alpha-3b/
```

```